tests/test_autonomy/test_cli/test_scaffold_fsm.py

```diff
@@ -52,12 +53,33 @@ class TestScaffoldFSM(AEATestCaseEmpty):
         "--spec",
     ]
 
+    @classmethod
+    def setup_class(cls) -> None:
+        """Set up the test class."""
+        super(AEATestCaseEmpty, cls).setup_class()
+        cls.agent_name = "default_author"
+        cls.create_agents(cls.agent_name, is_local=cls.IS_LOCAL, is_empty=cls.IS_EMPTY)
+        cls.set_agent_context(cls.agent_name)
+
     @pytest.mark.parametrize("fsm_spec_file", fsm_specifications)
     def test_run(self, fsm_spec_file: Path) -> None:
         """Test run."""
+
         my_skill = f"test_{fsm_spec_file.parts[-2]}"
         self.cli_options[-3] = my_skill
         path_to_spec_file = Path(ROOT_DIR) / fsm_spec_file
         args = [*self.cli_options, path_to_spec_file]
         result = self.run_cli_command(*args, cwd=self._get_cwd())
         assert result.exit_code == 0
+
+    def test_imports(self, monkeypatch: pytest.MonkeyPatch) -> None:
+        """Test imports of scaffolded modules"""
+
+        monkeypatch.syspath_prepend(self.t)
+        path = self.t / self.agent_name
+        for file in path.glob("**/*.py"):
+            if "tests" in file.parts:  # TODO
+                continue
+            spec = importlib.util.spec_from_file_location("name", file)
+            foo = importlib.util.module_from_spec(spec)
+            spec.loader.exec_module(foo)
```

Comment:
the nested test modules still contain erroneous imports (e.g. `from tests.test_skills.test_abstract_round_abci.test_tools.base import ...`), will be dealt with after the rest of the scaffolded skill is addressed